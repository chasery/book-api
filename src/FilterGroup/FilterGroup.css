.FilterGroup {
  display: flex;
  justify-content: center;
  padding: 0.5rem 0;
  background-color: lightgray;
}
