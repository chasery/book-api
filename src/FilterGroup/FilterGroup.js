import React from "react";
import Filter from "../Filter/Filter";
import "./FilterGroup.css";

class FilterGroup extends React.Component {
  render() {
    return (
      <div className="FilterGroup">
        <Filter />
        <Filter />
      </div>
    );
  }
}

export default FilterGroup;
