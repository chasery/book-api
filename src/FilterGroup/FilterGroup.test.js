import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import FilterGroup from "./FilterGroup";

describe("FilterGroup component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<FilterGroup />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
