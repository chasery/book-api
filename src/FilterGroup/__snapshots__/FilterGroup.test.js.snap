// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterGroup component renders without error 1`] = `
<div
  className="FilterGroup"
>
  <Filter />
  <Filter />
</div>
`;
